src/encoder/encoderopus.cpp

```diff
@@ -0,0 +1,465 @@
+// encoderopus.cpp
+// Create on August 15th 2017 by Palakis
+
+#include <stdlib.h>
+#include <QByteArray>
+#include <QMapIterator>
+#include <QtGlobal>
+
+#include "encoder/encoderopussettings.h"
+#include "engine/sidechain/enginesidechain.h"
+#include "util/logger.h"
+
+#include "encoder/encoderopus.h"
+
+// Opus only supports 48 and 96 kHz samplerates
+constexpr int EncoderOpus::MASTER_SAMPLERATE = 48000;
+
+const char* EncoderOpus::INVALID_SAMPLERATE_MESSAGE =
```

Comment:
The OPUS error message is hidden behind other error message (moved downwards in the screens for readability).

* When broadcasting:
The warning is displayed beneath the connection error message.
![capto_capture 2019-01-11_10-39-12_am](https://user-images.githubusercontent.com/4525897/51026126-1d24c880-158e-11e9-9f37-3dc2d4d62ce5.png)

* When recording:
  The warning is displayed beneath the recording error message, given no path is set in the recordings folder (default)
![capto_capture 2019-01-11_10-38-32_am](https://user-images.githubusercontent.com/4525897/51026121-19914180-158e-11e9-94e2-1a2252911a7b.png)